nacc/redcap2nacc.py

```diff
@@ -146,6 +146,69 @@ def check_for_bad_characters(field: Field) -> typing.List:
     return incompatible
 
 
+def check_redcap_event(options, record) -> bool:
+    """
+    Determines if the record's redcap_event_name and filled forms match the
+    options flag
+    """
+    if options.lbd and options.ivp:
+        event_name = 'initial_visit'
+        form_match_lbd = record['lbd_ivp_b1l_complete']
+        if (form_match_lbd == '0' or form_match_lbd == ''):
+            event_match = False
+            return event_match
+    elif options.lbd and options.fvp:
+        event_name = 'followup_visit'
+        form_match_lbd = record['lbd_fvp_b1l_complete']
+        if (form_match_lbd == '0' or form_match_lbd == ''):
+            event_match = False
+            return event_match
+    # TODO: add options for lbdsv (lbd short version)
+    elif options.ftld and options.ivp:
+        event_name = 'initial_visit'
+        form_match_ftld = record['ftld_ivp_a3a_complete']
+        if (form_match_ftld == '0' or form_match_ftld == ''):
+            event_match = False
+            return event_match
+    elif options.ftld and options.fvp:
+        event_name = 'followup_visit'
+        form_match_ftld = record['ftld_fvp_a3a_complete']
+        if (form_match_ftld == '0' or form_match_ftld == ''):
+            event_match = False
+            return event_match
+    # TODO: uncomment -csf option if/when it is added to the full ADRC project.
+    # Right now it is a single non-longitudinal form in a REDCap project with
+    # no redcap_event_names.
+    # elif options.csf:
+    #     event_name = 'csf_'
+    elif options.ivp:
+        event_name = 'initial_visit'
+        form_match_z1 = record['ivp_z1_complete']
+        form_match_z1x = record['ivp_z1x_complete']
+        if (form_match_z1 == '0' or form_match_z1 == '') and \
+                (form_match_z1x == '0' or form_match_z1x == ''):
```

Comment:
In Python, you can test multiple conditions like this:
```suggestion
        if form_match_z1 in ['', '0'] and form_match_z1x in ['', '0']:
```